server/src/http-exception.filter.ts

```diff
@@ -1,6 +1,25 @@
 import { ExceptionFilter, Catch, ArgumentsHost, HttpException } from '@nestjs/common';
 import { Request, Response } from 'express';
 
+
+function unfoldedStackTrace(response, status) {
```

Comment:
`export function unfoldedStackTrace` and unit test in the http-exception.filter test or somewhere

Reply:
Also, this is a really elegant and clear use of recursion. Really awesome job. Thank u for solving this!

Reply:
Thank you Matt!!